src/homework5/task1.py

```diff
@@ -0,0 +1,28 @@
+"""
+Оформите решение задач из прошлых домашних работ в функции.
+"""
+from src.homework5 import functions
+
+
+def run_function_from_module(module_name, func_name):
+    if hasattr(module_name, func_name):
+        func_instance = getattr(module_name, func_name)
+        if callable(func_instance):
+            print("Run func - ", func_name)
+            func_instance()
+        else:
+            print("Attribute [{name}] not callable".format(name=func_name))
+    else:
+        print("Func [{name}] not exist".format(name=func_name))
+
+
+def runner(*args):
+    if args:
+        [run_function_from_module(functions, param)
+         for param in args if isinstance(param, str)]
+    else:
+        [run_function_from_module(functions, param)
```

Comment:
I don't like your approach: 
I import module and run runner with no arg. It's expected behavior (it marked in task description). In fact in this case I will have a record in log (stdout) (Attribute [{name}] not callable) for each global variable in my imported module.

Reply:
not addressed